treeherder/log_parser/tasks.py

```diff
@@ -30,11 +30,7 @@ def parse_logs(job_id, job_log_ids, priority):
     if len(job_log_ids) != len(job_logs):
         logger.warning("Failed to load all expected job ids: %s", ", ".join(job_log_ids))
 
-    parser_tasks = {
-        "errorsummary_json": store_failure_lines,
-        "buildbot_text": parse_unstructured_log,
```

Comment:
buildbot_text looked like an old reference that was only found here and in a few tests.